### DISCUSSION TITLE
Github Actions Cron Job failure

### DISCUSSION THREAD

**Discussion by jilesingh** [community/community#122271] [Actions]
### Select Topic Area

Show & Tell

### Body

on:
  schedule:
    - cron: '33 11 * * *' 
    
hello everyone I am using cron job for github actions but my job is not runnign on the given time and it is giving me a delay of 5, 13, 22 minutes and then my scheduled job is running

**Comment by Anish570** [comment]
Hey there! It seems like your GitHub Actions job isn't running exactly on time, right? That could be due to a few factors, like server load or scheduling quirks. One common fix is to use a more precise scheduling method. Try using a more specific cron expression, like `33 11 * * *`, to ensure it runs exactly when you want. If you need more info on cron expressions, [check out this link](https://crontab.guru/). Hope that helps!

**Comment by n-thumann** [comment]
This is expected behavior: `The schedule event can be delayed during periods of high loads of GitHub Actions workflow runs` (see (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule). I observe the same behavior in one of my repositories, also with delays between 5 to 20 minutes.
